src/routes/health/routes.rs

```diff
@@ -3,7 +3,9 @@ use crate::cache::manager::ChainCache;
 use crate::utils::context::RequestContext;
 use crate::utils::errors::ApiResult;
 use rocket::response::content;
+use rocket_okapi::openapi;
 
+#[openapi(tag = "Health")]
```

Comment:
I'd exclude this from Swagger. It's used mostly by other services to check if the service is answering and not relevant for the clients

Reply:
No strong opinion on this one. I see this more as a tool for client side developers but I see your point. Let's leave it 👍